src/alda/Main.java

```diff
@@ -179,6 +183,7 @@ public static void main(String[] argv) {
     jc.addCommand("version", version);
 
     jc.addCommand("play", play);
+    jc.addCommand("stop", stop, "stop-playback");
```

Comment:
I'm not sure if we should put this command at the top level (ie: `alda stop`). 

Pros: It would be easy to type, short, and intuitive
Cons: It would clutter the top level with many commands (once we implement the rest of the playback controls). We would (eventually) have `alda stop`, `alda pause`, and `alda resume`.

I'm not sure if this is a  good idea, but we could have a `playback` command instead, so:

`alda playback stop/kill`, `alda playback resume/play`, and `alda playback pause`

This would also remove any ambiguity between the `alda play` command and the `alda resume` command if we ever get to that.

Both of the schemes seem fine to me, but it's important that we decide now so that we don't break backwards compatibility later. Also, we seem to be leaning more towards the existing behavior eg: `alda stop`, when looking at previous commands (ie: all the existing server control commands) so it's probably a good idea to keep going in that direction for now.